### ISSUE TITLE
cli: closed sessions seemingly left open

### ISSUE THREAD

**Issue by tiborsimko** [reanahub/reana-client#310]
When someone opens a Jupyter session, it works well, and the list reports:

```console
$ reana-client list --sessions
NAME               RUN_NUMBER   CREATED               SESSION_TYPE   SESSION_URI

root6-roofit.cwl   1            2019-08-08T09:47:39   jupyter        https://192.168.39.251:30443/4059f4da-0b2f-412b-9b5d-2ca16f5f0ff4?token=gSBPKWAKW988alPpd
n7piQwR6Fjdb3aj_3NTkHTUglk
```

When one is done with the notebook, and closes the session as follows:

```console
$ reana-client close -w root6-roofit.cwl
```

then the interactive pod gets well terminated.   However, the session is seemingly left open, as it were:

```console
$ reana-client list --sessions
NAME               RUN_NUMBER   CREATED               SESSION_TYPE   SESSION_URI
root6-roofit.cwl   1            2019-08-08T09:47:39   jupyter        https://192.168.39.251:30443None?token=gSBPKWAKW988alPpdn7piQwR6Fjdb3aj_3NTkHTUglk
```

Note the injected `None` in the URI.  

TL;DR The interactive pods are well closed, but listing of all open sessions still reports them as seemingly open (with wrong URI).

**Comment by tiborsimko** [comment]
Confirmed that this is still the case e.g. for PROD deployment.
